### YamlMime:ManagedReference
items:
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  id: RuntimeBinderException
  children:
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: RuntimeBinderException
  nameWithType: RuntimeBinderException
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  type: Class
  summary: "Representa um erro que ocorre quando uma associação dinâmica do associador de tempo de execução c# é processada."
  remarks: "Exceções desse tipo são diferentes de <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>nessa RuntimeBinderException representa uma falha ao associar no sentido de um erro de compilador normal, enquanto <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>representa um com mau funcionamento do associador de tempo de execução em si.</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException> </xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderInternalCompilerException>"
  syntax:
    content: 'public class RuntimeBinderException : Exception'
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  id: '#ctor'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException()
  nameWithType: RuntimeBinderException.RuntimeBinderException()
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException()
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Inicializa uma nova instância do <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> classe."
  remarks: "Este construtor inicializa o <xref:System.Exception.Message%2A>propriedade da nova instância para uma mensagem fornecida pelo sistema que descreve o erro, como &quot;DefaultMessageDisplayedByParameterlessCtorWriterMustSupply&quot; Esta mensagem leva em conta a cultura atual do sistema.</xref:System.Exception.Message%2A>       A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:System.InsufficientMemoryException>.</xref:System.InsufficientMemoryException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A cadeia de caracteres de mensagem de erro localizada. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuntimeBinderException ();
    parameters: []
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(String)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Inicializa uma nova instância do <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> classe que tem uma mensagem de erro especificada."
  remarks: "A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>.</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A cadeia de caracteres de mensagem de erro especificada no `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuntimeBinderException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem que descreve a exceção. O chamador desse construtor é necessário para garantir que essa cadeia de caracteres foi localizada para a cultura atual do sistema."
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(SerializationInfo,StreamingContext)
  nameWithType: RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Inicializa uma nova instância do <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> classe que tem dados serializado."
  remarks: "Este construtor é chamado durante a desserialização para reconstituir o objeto de exceção transmitido em um fluxo."
  syntax:
    content: protected RuntimeBinderException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "O objeto que contém os dados do objeto serializado sobre a exceção que está sendo gerada."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "As informações contextuais sobre a origem ou destino."
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  langs:
  - csharp
  name: RuntimeBinderException(String,Exception)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String,Exception)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String,Exception)
  type: Constructor
  assemblies:
  - Microsoft.CSharp
  namespace: Microsoft.CSharp.RuntimeBinder
  summary: "Inicializa uma nova instância do <xref href=&quot;Microsoft.CSharp.RuntimeBinder.RuntimeBinderException&quot;> </xref> classe que tem uma mensagem de erro especificada e uma referência à exceção interna que é a causa dessa exceção."
  remarks: "Uma exceção que é lançada como um resultado direto de uma exceção anterior deve incluir uma referência à exceção anterior no <xref:System.Exception.InnerException%2A>propriedade.</xref:System.Exception.InnerException%2A> O <xref:System.Exception.InnerException%2A>propriedade retorna o mesmo valor que é passado para o construtor, ou `null` se o <xref:System.Exception.InnerException%2A>propriedade não fornecer o valor da exceção interna para o construtor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>.</xref:Microsoft.CSharp.RuntimeBinder.RuntimeBinderException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A cadeia de caracteres de mensagem de erro especificada no `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuntimeBinderException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem de erro que explica o motivo da exceção."
    - id: innerException
      type: System.Exception
      description: "A exceção que é a causa da exceção atual ou uma referência nula se nenhuma exceção interna for especificada."
  overload: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException()
  nameWithType: RuntimeBinderException.RuntimeBinderException()
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException()
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(String)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(SerializationInfo,StreamingContext)
  nameWithType: RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor(System.String,System.Exception)
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException(String,Exception)
  nameWithType: RuntimeBinderException.RuntimeBinderException(String,Exception)
  fullName: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.RuntimeBinderException(String,Exception)
- uid: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException.#ctor*
  parent: Microsoft.CSharp.RuntimeBinder.RuntimeBinderException
  isExternal: false
  name: RuntimeBinderException
  nameWithType: RuntimeBinderException.RuntimeBinderException
